cli/src/status.rs

```diff
@@ -13,15 +13,37 @@ pub(crate) struct Status {
     /// Check the status of the testsys controller
     #[clap(long, short = 'c')]
     controller: bool,
+
+    /// Include the status of resources when reporting status
+    #[clap(long, short = 'p')]
+    progress: bool,
+
+    /// Include the `Test` status, too. Requires `--progress`
+    #[clap(long, short = 't', requires("progress"))]
+    with_test: bool,
+
+    /// Include the time the CRD was last updated
+    #[clap(long, short = 'u')]
+    with_time: bool,
```

Comment:
The options here are kind of hard to follow. `--progress/-p` shows status for resources, but not tests, for that you need `--with-test/-t`. If you want the CRD update time, you have to include `-u` which doesn't seem that intuitive.

I wonder if we could adopt similar to `kubelet get`'s `-o` option where `json` means json formatted output and `wide` means everything in a table. For custom tables they allow it through https://kubernetes.io/docs/reference/kubectl/#custom-columns. 

Let's not block on implementing that but I think we should just output the test object status by default since we care about that the most when kicking off tests and include a `--wide` option that outputs everything.

Reply:
I like that pattern!